decidim-proposals/spec/commands/decidim/proposals/endorse_proposal_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+module Decidim
+  module Proposals
+    describe EndorseProposal do
+      let(:proposal) { create(:proposal) }
+      let(:current_user) { create(:user, organization: proposal.feature.organization) }
+
+      describe "User endorses Proposal" do
+        let(:command) { described_class.new(proposal, current_user) }
+
+        context "when in normal conditions" do
+          it "broadcasts ok" do
+            expect { command.call }.to broadcast(:ok)
+          end
+
+          it "creates a new endorsement for the proposal" do
+            expect do
+              command.call
+            end.to change { ProposalEndorsement.count }.by(1)
+          end
+        end
+
+        context "when the endorsement is not valid" do
+          before do
+            # rubocop:disable RSpec/AnyInstance
```

Comment:
Can you try to fix this, please? It's better if we avoid skipping particular lint rules while adding new code.

Something like this could work (might need to be adapted):

```ruby
let(:endorsement) { instance_double(Decidim::Proposals::ProposalEndorsement, valid?: false) }

before do
  allow(ProposalEndorsement)
    .to receive(:where)
    # .with(hash_matching(key, value) # if needed
    .and_return([endorsement])
end
```

Same for all other cases.